README.md

```diff
@@ -28,11 +28,13 @@ TDR Jenkins has been configured to use the library functions with [Docker](https
 ## Testing functions on Jenkins
 
 1. Create a branch with the new function(s) on in the tdr-jenkinslib repo;
-2. In the Jenkins file that calls the new function(s) add the branch import directly: `@Library("tdr-jenkinslib@[name of branch]") _` 
-3. Create a test multi-branch pipeline job in Jenkins
-4. In the pipeline config add the library config to the Pipeline Libraries. Set the default to the name of the branch with the function(s) to test.
+2. In the Jenkins file that calls the new function(s) add the branch import directly: `@Library("tdr-jenkinslib@name-of-branch") _`
+3. Create a test pipeline job in Jenkins
+4. If the project is a multibranch pipeline, in the pipeline config add the library config to the Pipeline Libraries. Set the default to the name of the branch with the function(s) to test.
 5. When you replay a branch on the pipeline, all the code from the Jenkins file AND the library is available for editing
 
+If the job is a regular pipeline job rather than a multibranch pipeline, you can still test the Jenkinslib branch with `@Library("tdr-jenkinslib@name-of-branch") _`, but you cannot edit the library when replaying the build. You will have to make changes on the Jenkinslib branch and push them.
```

Comment:
You can potentially create a new test multibranch pipeline using the Jenkins script, and test it that way. It's a little cumbersome, but does provide a way to test a regular pipeline script.

Reply:
How's this? I've added the multibranch pipeline as an alternative, but said that you should filter it to just your own branch. (Otherwise the new job will try to run against all the open branches at once. If it's a deployment job, that might mean it tries to deploy all of those branches to intg.)